mlrun/datastore/sources.py

```diff
@@ -262,3 +265,5 @@ class HttpSource(OnlineSource):
     "http": HttpSource,
     "custom": CustomSource,
 }
+
+use_async_source = False
```

Comment:
Move to config - create a new datastore section and add there `async_source_mode` currently support `enabled`/`disabled` (but leaving room for more options)